Source/Core/Core/HLE/HLE_OS.cpp

```diff
@@ -2,19 +2,21 @@
 // Licensed under GPLv2+
 // Refer to the license.txt file included.
 
+#include <memory>
 #include <string>
 
 #include "Common/CommonTypes.h"
 #include "Common/Logging/Log.h"
 #include "Common/MsgHandler.h"
 #include "Common/StringUtil.h"
 #include "Core/HLE/HLE_OS.h"
+#include "Core/HLE/HLE_VarArgs.h"
 #include "Core/HW/Memmap.h"
 #include "Core/PowerPC/PowerPC.h"
 
 namespace HLE_OS
 {
-std::string GetStringVA(u32 strReg = 3);
+std::string GetStringVA(u32 strReg = 3, bool has_va_list = false);
```

Comment:
Not sure about how readable this would be... On top of being a default parameter, using a bool makes it hard to tell what the second parameter means without checking GetStringVA's signature. I'd consider using an enum class here.